homeassistant/components/peco/sensor.py

```diff
@@ -58,27 +64,33 @@ class PecoOutageCounterSensorEntity(CoordinatorEntity, SensorEntity):
     """PECO outage counter sensor."""
 
     _attr_state_class = SensorStateClass.MEASUREMENT
-    _attr_icon = "mdi:power-plug-off"
-
-    def __init__(self, hass, name, county, coordinator):
+    _attr_icon: str = "mdi:power-plug-off"
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        name: str,
+        county: str,
+        coordinator: DataUpdateCoordinator,
+    ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
-        self.hass = hass
-        self._county = county
-        self._name = name
+        self.hass: HomeAssistant = hass
+        self._county: str = county
+        self._name: str = name
```

Comment:
the annotations of the function attributes are inferred here, no need for duplication.